views/dashboard.pug

```diff
@@ -1,5 +1,7 @@
 extends ./layout
 
+block scripts
```

Comment:
I think you should use `append scripts` because it can override all scripts in `layout.pug`.
jQuery and Bootstrap.js not loaded on dashboard because of this.